server/views/pages/prisonerList/partials/searchTable.njk

```diff
@@ -28,7 +28,7 @@
             <td class="govuk-table__cell">{{ prisoner.releaseDate | formatDate('D MMMM YYYY') or 'N/A' }}</td>
             <td class="govuk-table__cell">{{ prisoner.receptionDate | formatDate('D MMMM YYYY') or 'N/A' }}</td>
             <td class="govuk-table__cell">
-              {% if prisoner.hasActionPlan !== true %}
+              {% if not (prisoner.hasCiagInduction === true and prisoner.hasActionPlan === true) %}
```

Comment:
Needs Plan should be shown if the prisoner either has no CIAG Induction or has no Action Plan.
Only if the prisoner has both do we not show the tag 👍